### ISSUE TITLE
includes is not a function on IE11

### ISSUE THREAD

**Issue by FredericHeem** [bvaughn/react-virtualized#419]
On IE11, *includes* in  not a function:
```
var scrollListener = function scrollListener(e) {
    // Don't measure (which forces) reflow for scrolls that happen inside of children!
    if (!e.target.className.includes('contract-trigger') && !e.target.className.includes('expand-trigger')) {
      return;
    }
``` 

IE outputs the error: `Object doesn't support property or method 'includes'`

**Comment by knpwrs** [comment]
You should probably use [`babel-polyfill`](https://www.npmjs.com/package/babel-polyfill) or [`core-js`](https://www.npmjs.com/package/core-js) if you want to support older browsers.

**Comment by bvaughn** [comment]
Blah. I get so used to working with application code that's transpiled+polyfilled that I don't think about IE sometimes. This could just as easily be `indexOf` so I'll change it.

**Comment by bvaughn** [comment]
The above change has been published in 8.0.10
